sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/EventProcessor.cs

```diff
@@ -206,6 +206,12 @@ internal EventProcessorStatus Status
         ///
         protected EventHubsRetryPolicy RetryPolicy { get; }
 
+        /// <summary>
+        ///    The properties associated with the Event Hub being read from. This value is updated in each load balancing cycle.
+        /// </summary>
+        ///
+        protected EventHubProperties Properties { get; private set; }
```

Comment:
```suggestion
        protected EventHubProperties EventHubProperties { get; private set; }
```

Reply:
Is it weird at all to have it have the same name as the type?

Reply:
Not so much; it's a pattern that we often use, though it usually does feel a bit odd.